### DISCUSSION TITLE
Reverse the display of facet menus

### DISCUSSION THREAD

**Discussion by britisharmy** [algolia/instantsearch#4818] [Q&A] [answered]
I have an app that is using instantsearch and some of the menus look like this

[![enter image description here][1]][1]

This is okay although the on position looks like this

[![enter image description here][2]][2]

which is taking more space. I want to summarize the facets to look like so

[![enter image description here][3]][3]

  [1]: https://i.stack.imgur.com/5mpvU.png
  [2]: https://i.stack.imgur.com/DUZJq.png
  [3]: https://i.stack.imgur.com/nfmD5.png

The refinement widget code looks like this

 <ais-toggle-refinement
	  attribute="free_cancellation"
	  label="Free Cancellation"
	   :on="0"
       :off="free_cancellation"
	    />

and this doesn't offer options for direct customizations.

How can i customize this widget to only have the checkbox, label text and count similar to the image attached?

**Comment by Haroenv** [comment, marked as answer]
You can use the `item` template to display less information in the refinementList, documentation for that is here: 

https://www.algolia.com/doc/api-reference/widgets/refinement-list/js/#widget-param-item

To completely customise the look of the widget, you can also look at the connector, docs for that are here:

https://www.algolia.com/doc/api-reference/widgets/refinement-list/js/#connector

**Comment by britisharmy** [comment]
@Haroenv Can `ais-menu` https://www.algolia.com/doc/api-reference/widgets/menu/vue/ have a checkbox at the start and count floated right ?. 

I have this

```
<ais-menu attribute="property_district">
		 <ul
			slot-scope="{
			  items,
			  canToggleShowMore,
			  isShowingMore,
			  toggleShowMore,
			  refine,
			  createURL,
			  sendEvent,
			}"
		  >
			<li v-for="item in items" :key="item.value">
			
			  <a
				:href="createURL(item.value)"
				:style="{ fontWeight: item.isRefined ? 'bold': '' }"
				@click.prevent="refine(item.value)">
				 <input type="checkbox" style="margin-right:2% !important;"/> {{item.label}} 
			  </a>
			  <span class="float-right pull-right">{{item.count}}</span>
			</li>
		  </ul>
		</ais-menu>
```

But the checkbox is not checkable. I attached change event

```
<ais-menu attribute="property_district">
		 <ul
			slot-scope="{
			  items,
			  canToggleShowMore,
			  isShowingMore,
			  toggleShowMore,
			  refine,
			  createURL,
			  sendEvent,
			}"
		  >
			<li v-for="item in items" :key="item.value">
			<input type="checkbox" @change="refine(item.value)" style="margin-right:2% !important;"/>
			  <a
				:href="createURL(item.value)"
				:style="{ fontWeight: item.isRefined ? 'bold': '' }"
				@click.prevent="refine(item.value)">
				  {{item.label}} 
			  </a>
			  <span class="float-right pull-right">{{item.count}}</span>
			</li>
		  </ul>
		</ais-menu>
```

and that works.

**Reply by Haroenv** [reply]
Yes, that's correct, in Vue InstantSearch the event doesn't automatically get applied, you need to attach refine manually like done in the second code sample!
